src/feature/hs/hs_config.c

```diff
@@ -493,9 +425,6 @@ config_service_v3(const config_line_t *line_,
 
   return 0;
  err:
-  if (dup_opt_seen) {
-    log_warn(LD_CONFIG, "Duplicate directive %s.", dup_opt_seen);
-  }
```

Comment:
So I suppose that duplicate option are found by the config subsystem? Else, we just removed that entirely, that is fine?

Reply:
Right. Duplicate options aren't allowed by default.